tests/test_api_requestor.py

```diff
@@ -232,14 +232,17 @@ def setup_stripe(self):
             "api_key": stripe.api_key,
             "api_version": stripe.api_version,
             "enable_telemetry": stripe.enable_telemetry,
+            "default_http_client": stripe.default_http_client,
         }
         stripe.api_key = "sk_test_123"
         stripe.api_version = "2017-12-14"
         stripe.enable_telemetry = False
+        stripe.default_http_client = None
         yield
         stripe.api_key = orig_attrs["api_key"]
         stripe.api_version = orig_attrs["api_version"]
         stripe.enable_telemetry = orig_attrs["enable_telemetry"]
+        stripe.default_http_client = orig_attrs["default_http_client"]
```

Comment:
In this instance and the two above it, could you make it so that `default_http_client` is inserted into this list alphabetically?

Reply:
Sure thing!